acoustid/future/fpindex/client.py

```
@@ -0,0 +1,396 @@
+            True if fingerprint exists, False otherwise
+        """
+        status, _ = await self._request(
+            "HEAD",
+            f"/{index_name}/{fingerprint_id}",
+            expected_status=[200, 404],
+            expected_response=None,
+        )
+        return status == 200
+
+    async def get_fingerprint_info(
+        self, index_name: str, fingerprint_id: int
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)